example/nodejs/Dockerfile

```diff
@@ -1,10 +1,8 @@
 ARG DOCKER_PROXY_HOST="docker.io"
 FROM ${DOCKER_PROXY_HOST}/node:16
 
-RUN mkdir -p /example/nodejs
```

Comment:
this is not needed, because of `WORKDIR $HOME` section creates `/example/nodejs` directory.